cdap-distributions/src/etc/cdap/conf.dist/cdap-site.xml.example

```diff
@@ -17,119 +17,152 @@
   -->
 <configuration>
 
-  <!--
-    Cluster configurations
-  -->
+  <!-- General Configuration -->
+
+  <property>
+    <name>hdfs.namespace</name>
+    <value>/${root.namespace}</value>
+    <description>
+      Root directory for HDFS files written by CDAP
+    </description>
+  </property>
+  
+  <property>
+    <name>hdfs.user</name>
+    <value>yarn</value>
+    <description>
+      User name for accessing HDFS
+    </description>
+  </property>
 
   <property>
     <name>root.namespace</name>
     <value>cdap</value>
-    <description>Specifies the root namespace</description>
+    <description>
+      Root for this CDAP instance; used as the parent (or root) node for
+      ZooKeeper, as the directory under which all CDAP data and metadata is
+      stored in HDFS, and as the prefix for all HBase tables created by
+      CDAP; must be composed of alphanumeric characters
+    </description>
   </property>
 
-  <!-- Substitute the zookeeper quorum for components here -->
   <property>
     <name>zookeeper.quorum</name>
     <value>FQDN1:2181,FQDN2:2181/${root.namespace}</value>
-    <description>Specifies the zookeeper host:port</description>
+    <description>
+      ZooKeeper quorum string; specifies the ZooKeeper host:port; 
+      substitute the quorum for the components shown here (FQDN1:2181,FQDN2:2181)
+    </description>
   </property>
 
+
+  <!-- Router Configuration -->
+
   <property>
-    <name>hdfs.namespace</name>
-    <value>/${root.namespace}</value>
-    <description>Namespace for HDFS files</description>
+    <name>router.bind.address</name>
+    <value>0.0.0.0</value>
+    <description>
+      Router server address
```

Comment:
probably out of scope for this PR, but we shouldn't have the description of one property be the name of another property :)